uplink/src/bucket.rs

```diff
@@ -3,20 +3,20 @@
 use crate::uplink_c::Ensurer;
 use crate::{Error, Result};
 
-use std::ffi::CStr;
+use std::ffi::{CStr, CString};
 use std::time::Duration;
 
 use uplink_sys as ulksys;
 
 /// Contains information about a specific bucket.
-pub struct Bucket<'a> {
+pub struct Bucket {
     /// Name of the bucket.
-    pub name: &'a str,
+    pub name: String,
```

Comment:
changing the lifetime awareness is fine.  may be sub-optimal in the future, however, easier to read now.